### PR TITLE
Added CSRF protection with plone.protect.

### PR DESCRIPTION
**Pull request by phgross**
The CSRF protection will only work if plone.protect is in the version 2.0.2 or higher (see https://github.com/plone/plone.protect/blob/2.0.2/plone/protect/__init__.py#L5 for more details).

@jone @lukasgraf please have a look ...

### CODE REVIEW

File: ftw/dashboard/dragndrop/browser/dashboard.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
-from Products.CMFCore.utils import getToolByName
 from plone.app.layout.dashboard.dashboard import DashboardView
 from plone.portlets.interfaces import IPortletType
+from Products.CMFCore.utils import getToolByName
 from zope.component import queryUtility
 
 
+try:
```
Comment (phgross) [inline ftw/dashboard/dragndrop/browser/dashboard.py]: Is there a better solution to check if the `createToken` method is available i.e. it's in version `2.0.2` or higher?

Comment (jone) [inline ftw/dashboard/dragndrop/browser/dashboard.py]: This is ok, but I'd maybe add a comment..

File: ftw/dashboard/dragndrop/browser/resources/dnd_dashboard.js [hunk 2]
```diff
@@ -90,12 +92,20 @@ jQuery(function($){
       type :      'POST',
       url :       './ftw.dashboard.dragndrop-update_order',
       data :      customSerialization(),
+      beforeSend: function (request)
+      {
```
Comment (jone) [inline ftw/dashboard/dragndrop/browser/resources/dnd_dashboard.js]: I'd move the `{` to the end of the line above to preserve local consistency. Same at the other places..

**Comment by jone** [comment]
:+1:

**Comment by phgross** [comment]
@ Added comment in import and cleaned up js formatting ...

File: ftw/dashboard/dragndrop/browser/dashboard.py (hunk 1 above)
Comment (phgross) [inline ftw/dashboard/dragndrop/browser/dashboard.py]: ah pep8 prefer 

```python
try:
    from plone.protect import createToken
except ImportError:
    def createToken(): return ''
```

i will change that ...

File: ftw/dashboard/dragndrop/browser/resources/dnd_dashboard.js [hunk 3]
```diff
@@ -34,6 +34,8 @@ jQuery(function($){
     }
   };
 
+  var authenticator_token = $('.dashboardContent').data('authenticator-token')
```
Comment (jone) [inline ftw/dashboard/dragndrop/browser/resources/dnd_dashboard.js]: please add a trailing semicolon :wink:

**Comment by phgross** [comment]
@jone updated ...

**Pull request merged**